packages/client/package.json

```diff
@@ -12,7 +12,7 @@
     "build": "VITE_APP_VERSION=$VERSION NODE_OPTIONS=--max_old_space_size=8000 vite build",
```

Comment:
## Client
The most significant change is that the data model for user.role looks now a little different. Instead of a `labels` array with a single entity label for each supported language, the client now only receives a singular `label`, the is a message descriptor object. This object needs to be passed through `react-intl` to get the end-user friendly string.